apps/datahub/src/app/router/datahub-router.service.ts

```diff
@@ -29,16 +29,21 @@ export class DatahubRouterService {
     return [
       {
         path: '',
-        redirectTo: `${ROUTER_ROUTE_HOME}/${ROUTER_ROUTE_NEWS}`,
+        redirectTo: ROUTER_ROUTE_NEWS,
         pathMatch: 'full',
       },
       {
         path: ROUTER_ROUTE_HOME,
```

Comment:
There are 2 redirections for `ROUTER_ROUTE_HOME`, one should be useless then.

Reply:
The one that redirect to news is, in sort of, useless. It's just in the case if someone hit the `/home/` path.  I don't know if there will be some redirection to this old path from other websites/softwares, but let it in case it does. 
Should I remove it ?

Reply:
IMO yes cause nobody goes to `/home`, either they went to `/` or `/home/news`

Reply:
/Home removed